deploy/cr.yaml

```diff
@@ -33,7 +33,7 @@ spec:
   mysql:
     clusterType: group-replication
     autoRecovery: true
-    image: percona/percona-server:8.0.36-28
+    image: perconalab/percona-server-mysql-operator:main-psmysql
```

Comment:
Please change following things in cr.yaml to 0.8.0 as well:
```
$ grep -R "0\.7\.0" .
./cr.yaml:  crVersion: 0.7.0
./cr.yaml:#    initImage: percona/percona-server-mysql-operator:0.7.0
./cr.yaml:#      initImage: percona/percona-server-mysql-operator:0.7.0
./cr.yaml:#    initImage: percona/percona-server-mysql-operator:0.7.0
./cr.yaml:#    initImage: percona/percona-server-mysql-operator:0.7.0
```

Reply:
done